person.rb

```diff
@@ -0,0 +1,25 @@
+require 'securerandom'
+
+class Person
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = SecureRandom.uuid
```

Comment:
- [ ] [OPTIONAL] You can give UUID a range. Kind of:


```suggestion
    @id = Random.rand(1..1000)
```